src/collection.js

```diff
@@ -470,6 +470,18 @@ class Collection {
     return mapped.some(value => value)
   }
 
+  /**
+   * Returns a sorted list of all collection items, with an optional comparator
+   *
+   * @param {Function} comparator
+   *
+   * @returns {Collection}
+   */
+  async sort (comparator) {
+    this.items = [...this.items.sort(comparator)]
+    return this

```

Comment:
Can you please add an empty line before the return statement?

Reply:
just pushed the formatting change